### DISCUSSION TITLE
Missing `raise` on exception

### DISCUSSION THREAD

**Discussion by Kludex** [astral-sh/ruff#10165] [Q&A]
Hi there,

Not sure if it's even feasible or make sense... But is there a rule for exception that is not being raised?

```py
PotatoError()  # ruff should error
raise PotatoError()  # ruff should NOT error
```

I don't think there exists given that ruff doesn't know that `PotatoError()` is an exception at that point (right?), but asking just in case...

**Comment by UnknownPlatypus** [comment]
This could be flagged for builtin exceptions at least ?
